src/glass/src/app/pages/login-page/login-page.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="glass-login-page">
+  <div class="glass-login-page-background"></div>
+  <div class="glass-login-page-content"
+       fxLayout="column">
+    <mat-card>
+      <mat-card-content>
+        <glass-declarative-form [config]="config"></glass-declarative-form>
+      </mat-card-content>
+      <mat-card-actions fxLayoutAlign="center center">
+        <glass-submit-button [form]="form?.formGroup"
+                             (buttonClick)="onLogin()">
```

Comment:
Would it be possible to call `doLogin()` by hitting 'enter' after entering the password? Just a minor thing. It shouldn't block the PR from being merged and can be addressed in a different issue if you prefer.

Reply:
No, that's not possible because the submit button is not part of the form. If this is required, we need to refactor the form implementation.

Reply:
I'll check that, didn't find the reason why it does not work till now.

Reply:
Fixed

Reply:
Nice :partying_face: Thank you!